src/main.ts

```diff
@@ -140,7 +142,7 @@ const executeDockerCommand = async (dockerCommand: string, continueOnError: bool
     };
     var exitCode;
     try {
-        exitCode = await exec.exec(`"${dockerTool}" ${dockerCommand}`, [], execOptions);
+        exitCode = await exec.exec(`"${dockerTool}" ${dockerCommand}`, args, execOptions);

```

Comment:
(At least for Python `subprocess` module) The convention is to only use `commandLine` to specify the docker tool's path `${dockerTool}`. `${dockerCommand}` such as `run` should be included in `args`. Even though they are docker's commands, on Shell level, they are command's arguments.

https://docs.python.org/3/library/subprocess.html#subprocess.Popen

> On POSIX, if args is a string, the string is interpreted as the name or path of the program to execute. However, this can only be done if not passing arguments to the program.